src/lib/components/SiteFooter/index.jsx

```diff
@@ -350,6 +404,13 @@ SiteFooter.defaultProps = {
   showMobileLinks: true,
   appleAppStoreUrl: null,
   googlePlayUrl: null,
+  supportedLanguages: [],
+  languageForm: {
+    screenReaderLabel: 'Choose Language',
+    submitLabel: 'Apply',
+    onLanguageSelected: () => {},
+    activeLanguage: 'en',
```

Comment:
This should be read from their cookie, right?  Or is that done somewhere else and I'm missing it?

Reply:
I'm purposely keeping that logic separate from this component by using props.